add-on/TA-Demisto/README.md

```diff
@@ -21,16 +19,57 @@ Supporting Add-on for Demisto. This application allows a user to create incident
+
+2) Add the certificate to the app installation tgz:
+    2.1) Extract TA-Demisto tgz
+    2.2) Create "local" directory under TA-Demisto
+    2.3) Place the certificate in that folder
+    In the end the certificate should be in the following path:
+    your-local-path/TA-Demisto/local/cert_bundle.pem
+
+Splunk Cloud users should perform option 2 and send the app installer to Splunk support for installation.
 
-* User can directly extract SPL file  into $SPLUNK_HOME/etc/apps/ folder.
+Another option which applies ONLY for on-prem installations and is cannot be used for Splunk Cloud:
 
+Disabling the certificate validation entirely by POSTing to the Splunk REST API
+https://splunk-server:8089/servicesNS/nobody/TA-Demisto/configs/conf-demistosetup/demistoenv
+with the following as the request body:
+verify_ssl=false
+
+For example, via CURL:
+curl -ku 'username:password' https://localhost:8089/servicesNS/nobody/TA-Demisto/configs/conf-demistosetup/demistoenv/ -d VALIDATE_SSL=false
+
+To re-enable certificate validation post to the same endpoint but change "verify_ssl" to true.
+
+For example, via CURL:
+curl -ku 'username:password' https://localhost:8089/servicesNS/nobody/TA-Demisto/configs/conf-demistosetup/demistoenv/ -d VALIDATE_SSL=true
+
+We recommend to use certificates, and only disabling certificate verification in development
+or test environments only. Never disable certificate verification for a production system.
+
+* User can directly extract the app's SPL file into $SPLUNK_HOME/etc/apps/ folder in order to install the app.
 
 # Application Setup
-* The user must complete the setup of the application. In order to create incident into Demisto, a user needs to provide following four parameters:
+* The user must complete the setup of the application. In order to create incident into Demisto, a user needs to provide the following parameters:
     1)Demisto URL: This is mandatory parameter Url /IP address of the Demisto
     2)Demisto Port: This is an optional parameter. The user must define it if running Demisto on any other port than the default (443).
-    3)Allow Self Signed Certificate: User should select this if using Self Signed certificate
-    4) Authentication key: This is a mandatory parameter. This parameter is used for authorization with Demsito. In order to generate this parameter, a user should log in to Demisto and then click on Settings --> Integration --> API Keys.
+    3)Authentication key: This is a mandatory parameter. This parameter is used for authorization with Demisto. In order to generate this parameter,
+      a user should log in to Demisto and then click on Settings --> Integration --> API Keys.
+    4)Location to certificate: Enter the full path to the SSL Certificate in the Splunk server to if you are using Self Signed/Internal CA signed certificate
+    5)HTTPS Proxy Address: This is an optional parameter. Define this if you have HTTPS proxy that should be used. Should be in the format of https://ip:port
```

Comment:
There is a standard for proxy with creds. why don't we use it? 
`user:password@proxyIP:proxyPort`

Reply:
@orius123 we can change that part of the code in the next version.